### PR TITLE
Feature/add upload wopi

### PR DESCRIPTION
**Pull request by VolshebnikQ**

### CODE REVIEW

**Review by LinneyS — CHANGES_REQUESTED** [review]
```
ConverExtList is not defined
jscript.js:120
```

**Review by Eloncase — APPROVED** [review]

File: web/documentserver-example/nodejs/views/wopiIndex.ejs [hunk 1]
```diff
@@ -250,6 +300,14 @@
     <script type="text/javascript" src="javascripts/jquery.fileupload.js"></script>
     <script type="text/javascript" src="javascripts/jquery.dropdownToggle.js"></script>
     <script type="text/javascript" src="javascripts/jscript.js"></script>
+
+    <script type="text/javascript">
+        var ConverExtList = "<%= convertExts %>";
+        var EditedExtList = "<%= editedExts %>";
+        var FilledExtList = "<%= fillExts %>";
+        var UrlConverter = "convert";
+        var UrlEditor = "editor";
```
Comment (LinneyS) [inline web/documentserver-example/nodejs/views/wopiIndex.ejs:309]: Not correct for WOPI

File: web/documentserver-example/nodejs/views/wopiIndex.ejs [hunk 2]
```diff
@@ -223,6 +229,50 @@
...
+            <div class="error-message">
+                <b>Upload error: </b><span></span>
+                <br />
+                Please select another file and try again.
+            </div>
+        </div>
+        <br />
+        <div id="beginEdit" class="button orange disable">Edit</div>
+        <div id="beginView" class="button gray disable">View</div>
+        <div id="beginEmbedded" class="button gray disable">Embedded view</div>
```
Comment (LinneyS) [inline web/documentserver-example/nodejs/views/wopiIndex.ejs:270]: Remove

**Review by LinneyS — CHANGES_REQUESTED** [review]
dragdrop upload?

File: web/documentserver-example/nodejs/helpers/wopi/wopiRouting.js [hunk 3]
```diff
@@ -72,6 +72,10 @@ exports.registerRoutes = function(app) {
                 params: docManager.getCustomParams(),
                 users: users,
                 serverUrl: docManager.getServerUrl(),
+                preloaderUrl: siteUrl + configServer.get('preloaderUrl'),
+                convertExts: configServer.get('convertedDocs'),
+                editedExts: configServer.get('editedDocs'),
```
Comment (LinneyS) [inline web/documentserver-example/nodejs/helpers/wopi/wopiRouting.js:77]: Filter extensions from config that are in discovery for edit action

**Review by LinneyS — CHANGES_REQUESTED** [review]

File: web/documentserver-example/nodejs/public/javascripts/jscript.js [hunk 4]
```diff
@@ -198,8 +198,10 @@ if (typeof jQuery != "undefined") {
         var posExt = fileName.lastIndexOf('.');
         posExt = 0 <= posExt ? fileName.substring(posExt).trim().toLowerCase() : '';
 
-        if (EditedExtList.indexOf(posExt) != -1
-            || FilledExtList.indexOf(posExt) != -1) {
+        var checkEdited = EditedExtList.split(",").find(i => i == posExt);
+        var checkFilled = FilledExtList.split(",").find(i => i == posExt);
```
Comment (LinneyS) [inline web/documentserver-example/nodejs/public/javascripts/jscript.js:202]: support IE required

**Review by LinneyS — CHANGES_REQUESTED** [review]

**Pull request merged**